src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
@FibreFoX That's OK with me. I'm just hardcoding the properties names, so I hope they won't change too much.
https://github.com/Drombler/drombler-fx/blob/137-packaging-support/drombler-fx-maven-plugin/src/main/java/org/drombler/fx/maven/plugin/util/JavaFXMavenPluginUtils.java
https://github.com/Drombler/drombler-fx/blob/137-packaging-support/drombler-fx-maven-plugin/src/main/java/org/drombler/fx/maven/plugin/SetMavenProperties.java

Currently I'm using the Maven Properties to integrate the JavaFX Maven Plugin as programmatically manipulating the plugin configuration doesn't seem to have any effect (the changes get lost).